CONTRIBUTING.md

```diff
@@ -47,130 +105,147 @@ the [Software Carpentry introduction to git][link_swc_intro].
-
-```Shell
-git fetch upstream
-git checkout master
-git merge upstream/master
-```
+| Label                                                                                           | Description                                                                                           |
+|--------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------|
+| ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20high) <br> ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20medium) <br> ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20low)    | Estimation of the downstream impact the proposed feature/bugfix will have.                |
+| ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20high) <br> ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20medium) <br> ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20low)    | Estimation of effort required to implement the requested feature or fix the reported bug. |
 
-**4. Make the changes you've discussed, following the [fMRIPrep coding style guide](#fMRIPrep-coding-style-guide).**
+These labels help triage and set priorities to the development tasks.
+For instance, one bug regression that has been reported to affect most of the users after
+a release with an easy fix because it is a known old problem that came back.
+Such an issue will typically be assigned the following labels ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/bug) ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20high) ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20low), and its priority will be maximal since addressing low-effort high-impact deliver the maximum turnout without increasing the churn by much.
 
-Try to keep the changes focused.
-If you feel tempted to "branch out" then please make a [new branch][link_branches].
-It can also be helpful to test your changes locally,
-using an [fMRIPrep development environment][link_devel].
-Once you are satisfied with your local changes, [add/commit/push them][link_add_commit_push]
-to the branch on your forked repository.
+Of course, the implementation of long-term goaled lines may include the scheduling of ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20medium) ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20high).
+Finally, ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20low) ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20high) issues are less likely to be addressed.
```

Comment:
```suggestion
Of note, issues with the labels:
 ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/impact%3A%20low) or ![GitHub labels](https://img.shields.io/github/labels/poldracklab/fmriprep/effort%3A%20high)
are less likely to be addressed because they are less likely to impact the code-base, or because they will require a very high activation energy to do so.
```